stuffer/s2n_stuffer.c

```diff
@@ -101,9 +102,8 @@ int s2n_stuffer_resize(struct s2n_stuffer *stuffer, const uint32_t size)
 int s2n_stuffer_rewrite(struct s2n_stuffer *stuffer)
 {
     stuffer->write_cursor = 0;
-    if (stuffer->read_cursor > stuffer->write_cursor) {
-        stuffer->read_cursor = stuffer->write_cursor;
-    }
+    stuffer->read_cursor = MIN(stuffer->read_cursor, stuffer->write_cursor);
```

Comment:
So this is MIN(stuffer->read_cursor, 0). Should we just eliminate the MIN and do `stuffer->write_cursor = 0; 
stuffer->read_cursor = 0;`